lib/perl/Genome/VariantReporting/Framework/Component/Expert/Command.pm

```diff
@@ -101,7 +185,7 @@ sub input_hash {
 
     $hash{test_name} = $ENV{GENOME_SOFTWARE_RESULT_TEST_NAME};
 
-    for my $key (qw(requestor user sponsor process_id label)) {
+    for my $key (qw(requestor user sponsor process_id label plan_json)) {
```

Comment:
Do we still need to delete these?  It seems like since we are limiting to is_argument maybe they won't be getting put in the hash?